src/com/google/devtools/intellij/ijaas/OpenFileManager.java

```diff
@@ -39,7 +44,11 @@
 
   @Nullable
```

Comment:
Now looking at the code, we don't use this method at all. Let's delete this getByPath